ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -418,20 +418,6 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
-- as: aws-ipi-disconnected-private-tp-f28-destructive
```

Comment:
Did you delete it on purpose? If no, suggest to keep it.

Reply:
deleted on purpose.

Reply:
@miyadav We may need keep this one as CAPI cases are destructive. We should keep one normal job and one destructive job for each platform, right? cc @sunzhaohua2

Reply:
Yes @miyadav there are 2 jobs for each workflow, we'd better to keep the 2 jobs

Reply:
@sunzhaohua2 we kept - `aws-ipi-disconnected-private-tp-f28`  and `azure-ipi-proxy-tp-f28` 
you guys want one destructive in place of one of these or destructive in addition to these ? Because in slack conversation we decided only these two earlier .But the rationale to have destructive is convincing . Let me know.

Reply:
@miyadav sorry for any confusion, let's kept aws-ipi-disconnected-private-tp-f28, aws-ipi-disconnected-private-tp-f28-destructive, azure-ipi-proxy-tp-f28 and azure-ipi-proxy-tp-f28-destructive